sqlglot/parser.py

```diff
@@ -660,6 +660,12 @@ def _advance(self, times=1):
     def _retreat(self, index):
         self._advance(index - self._index)
 
+    def _attach_comment(self, node, comment):
+        try:
+            node.comment = comment
+        finally:
+            return node
+
```

Comment:
This method is sometimes called from `parse_csv` with a tuple argument for `node`, so `node.column` fails. I thought that the try / finally approach is a good choice to handle any such scenario where we don't have the `comment` attribute and is better than `hasattr`, probably.